cmd/routed-eni-cni-plugin/cni_test.go

```diff
@@ -570,6 +571,7 @@ func Test_tryDelWithPrevResult(t *testing.T) {
 				conf: &NetConf{
 					NetConf: types.NetConf{
 						PrevResult: &current.Result{
+							CNIVersion: "1.0.0",
```

Comment:
This has to be "0.4.0"

https://github.com/aws/amazon-vpc-cni-k8s/blob/master/misc/10-aws.conflist#L2